src/views/ScopesetComparison/index.jsx

```diff
@@ -0,0 +1,203 @@
+  state = {
+    scopeTextA: '',
+    scopeTextB: '',
+  };
+
+  handleScopesAChange = scopeTextA => {
+    this.setState({ scopeTextA });
+  };
+
+  handleScopesBChange = scopeTextB => {
+    this.setState({ scopeTextB });
+  };
+
+  handleCompareScopesClick = async () => {
+    const { scopeTextA, scopeTextB } = this.state;
+
+    if (scopeTextA && scopeTextB) {
+      const scopesA = splitLines(scopeTextA);
+      const scopesB = splitLines(scopeTextB);
+      const scopesetDiff = this.getScopesetDiff(scopesA, scopesB);
+      const cellColors = this.getCellColors(scopesetDiff);
+
+      this.setState({ scopesetDiff, cellColors });
+    }
+  };
+
+  getScopesetDiff = (scopesA, scopesB) => {
+    const scopesUnion = scopeUnion(scopesA, scopesB);
+    const scopesetDiff = [];
+
+    scopesUnion.forEach(scope => {
+      const s1 = scopeIntersection([scope], scopesA);
+      const s2 = scopeIntersection([scope], scopesB);
+
+      scopesetDiff.push([s1, s2]);
+    });
+
+    return scopesetDiff;
+  };
+
+  getCellColors = scopesetDiff => {
+    const cellColors = [];
+    const CELLS = {
+      G: 'green',
+      R: 'red',
+      Y: 'yellow',
+    };
+
+    scopesetDiff.forEach(scope => {
+      if (scope[0].length === 0 && scope[1].length) {
+        cellColors.push(['', `${CELLS.G}Cell`]);
+      } else if (scope[0].length && scope[1].length === 0) {
+        cellColors.push([`${CELLS.R}Cell`, '']);
+      } else if (!equals(scope[0], scope[1])) {
+        cellColors.push([`${CELLS.Y}Cell`, `${CELLS.Y}Cell`]);
+      } else {
+        cellColors.push(['', '']);
+      }
+    });
+
+    return cellColors;
+  };
+
+  render() {
+    const { classes } = this.props;
+    const { scopeTextA, scopeTextB, scopesetDiff, cellColors } = this.state;
+
+    return (
+      <Dashboard title="Compare Scopesets">
+        <Fragment>
+          <Grid className={classes.editorGrid} container spacing={16}>
+            <Grid item xs={12} md={6}>
+              <Typography gutterBottom align="center" variant="subheading">
+                Scope A
+              </Typography>
+              <CodeEditor
+                className={classes.editor}
+                onChange={this.handleScopesAChange}
+                placeholder="new-scope:for-something:*"
+                mode="scopemode"
+                value={scopeTextA}
+              />
+            </Grid>
+            <Grid item xs={12} md={6}>
+              <Typography gutterBottom align="center" variant="subheading">
+                Scope B
+              </Typography>
+              <CodeEditor
+                className={classes.editor}
+                onChange={this.handleScopesBChange}
+                placeholder="new-scope:for-something:*"
+                mode="scopemode"
+                value={scopeTextB}
+              />
+            </Grid>
+          </Grid>
+          {scopesetDiff &&
+            cellColors && (
+              <Table>
```

Comment:
Replace table with the Grid component.

```jsx
<Grid container ..>
  <Grid item xs={6}>...</Grid>
  <Grid item xs={6}>...</Grid>
</Grid>
```

Reply:
The end result should look something like:

![screen shot 2018-10-01 at 3 40 06 pm](https://user-images.githubusercontent.com/3766511/46311574-ab051680-c590-11e8-87a0-0c52b0336d87.png)